api/spec/v1.yaml

```diff
@@ -767,17 +767,17 @@ components:
       properties:
         latest_chain_id:
           type: string
-          description: The latest chain ID at the head of indexing.
+          description: The most recently indexed chain ID.
           example: *chain_id_1
         latest_block:
           type: integer
           format: int64
-          description: The latest indexed block at the head of indexing.
+          description: The height of the most recent indexed block. Query a synced Oasis node for the latest block produced.
           example: *block_height_1
         latest_update:
           type: string
           format: date-time
-          description: The RFC 3339 formatted time of latest indexing update.
+          description: The RFC 3339 formatted time of the last Indexer update. Compare with current time for approximate indexing progress with the Oasis Network.
```

Comment:
wait was this to track indexer updates? (and is an indexer update what I think it is?)

Reply:
> indexer updates

This is the time of the processing of the most recent ingested block by the Indexer. 😓 

> indexer update

Oh boy. Uhh, maybe this is not clear. An indexer update is not related to deploy/versioning but ingest or block processing. The source of this information is:

https://github.com/oasisprotocol/oasis-indexer/blob/2cb4ddda6b521c98d59ef0b1746eee881ef7f462/storage/client/queries.go#L16-L22

We then infer that at `latest_update` timestamp. The Indexer fully processed its most recent block (or greatest height block).

Reply:
```suggestion
          description: The RFC 3339 formatted time at which the Indexer processed the latest block. Compare with current time for approximate indexing progress with the Oasis Network.
```

Reply:
Open to further future changes, but merging for now.